spec/fixtures/custom_nginx.template

```diff
@@ -312,60 +312,156 @@ http {
 
     server {
         listen 55555;
+        server_name mock_upstream;
 
-        location /ws {
+        location = / {
+            access_by_lua_block {
+                local mu = require('spec.fixtures.mock_upstream')
+                return mu.filter_access_by_method("GET")
+            }
             content_by_lua_block {
-                local server = require "resty.websocket.server"
-                local wb, err = server:new{
-                  timeout = 5000,
-                  max_payload_len = 65535,
-                }
-
-                if not wb then
-                  ngx.log(ngx.ERR, "failed to open websocket: ", err)
-                  return ngx.exit(444)
-                end
+                local mu = require('spec.fixtures.mock_upstream')
+                return mu.send_json_response({
+                    valid_routes = {
+                        ["/ws"] = "web sockets echo server",
+                        ["/get"] = "Accepts a GET request and returns it in json format",
+                        ["/post"] = "Accepts a POST request and returns it in json format",
+                        ["/anything"] = "Accepts any request and returns it in json format",
+                        ["/request"] = "Alias to /anything",
+                        ["/delay/s"] = "Delay the response for s seconds",
```

Comment:
Could we document the endpoint in such a way that the variable path segment is documented: `/delay/:s`?